transforms/universal/ededup/python/src/ededup_transform_python.py

```diff
@@ -35,8 +45,9 @@ def __init__(self, config: dict):
             doc_column - name of the doc column
         """
         super().__init__(config)
-        self.filter = HashFilter({})
-        self.stats = config.get("statistics", None)
+        self.filter = config.get("filter", None)
```

Comment:
I think we need a default here so the transform can more easily be run outside of the python runtime/launcher.

Reply:
there is no default. we can run it outside python. see example

Reply:
But this requires the call to provide "filter". why should we required this when it seems we have an implementation that is expected to be used?

Reply:
I still thing its wrong, but added this